[AT-1171] Add documentation for driver set up with DbVisualizer (#3)
Bit-Quill/amazon-timestream-driver-jdbc: markdown/setup/dbvisualizer-driver-setup.md

### DbVisualizer 
[Link to product webpage](https://www.dbvis.com/download/).

#### Adding the Amazon Timestream JDBC Driver
1. [Download](https://github.com/awslabs/amazon-timestream-driver-jdbc/releases/latest) the Timestream JDBC driver shaded JAR file (e.g., `amazon-timestream-jdbc-2.0.0-shaded.jar`)
2. Start the DbVisualizer application and navigate to the menu path: **Tools > Driver Manager...**. In the popped up window, click the "Custom" link on the right side.
>>![Example](../images/dbvisualizer/DbVisualizer1.png)

3. In the new popped up window, find **Driver Settings** tab and fill the following info.

>  >> a. For the **Name:** field, enter **Timestream**

Reviewer: ```suggestion
 >> a. For the **Name:** field, enter **Timestream**
 
```
new line is required for rendering

>  >> b. For the **URL Format:** field, enter ```jdbc:timestream://PropertyName1=value1;PropertyName2=value2...```

Reviewer: ```suggestion
 >> b. For the **URL Format:** field, enter ```jdbc:timestream://PropertyName1=value1;PropertyName2=value2...```. For a list of connection properties, see [README](../../README.md#optional-connection-properties).

```

Author: Done

>  >> c. Click the **+** on the right side to select the Timestream JDBC JAR file.

Reviewer: ```suggestion
 >> c. Click the **+** on the right side to select the Timestream JDBC JAR file.
 
```

>  >> d. Ensure the ```software.amazon.timestream.jdbc.TimestreamDriver``` is selected in the **Driver Class:** field. Your Driver Manager settings for **Timestream** should look like the following image.

Reviewer: ```suggestion
 >> d. Ensure the ```software.amazon.timestream.jdbc.TimestreamDriver``` is selected in the **Driver Class:** field. Your Driver Manager settings for **Timestream** should look like the following image.
 
```

 >>>![Example](../images/dbvisualizer/DbVisualizer2.png)

 >>e. Close the dialog. Now the Timestream JDBC driver is added and ready to use.

 4. Connecting to Amazon Timestream Using DbVisualizer

  >> a. Navigate the menu path **Database > Create Database Connection**

  >> b. For the **Name** field, enter a descriptive name for the connection, for example ```TimestreamConnection```

  >> c. For the **Driver Type** field, ensure the **Timesteam** is selected.

>   >> d. For the **Database URL** field, enter your JDBC connection string. For example, ```jdbc:timestream://region=us-east-1;```

Reviewer: ```suggestion
  >> d. For the **Database URL** field, enter your [JDBC connection string](../../README.md#optional-connection-properties). For example, ```jdbc:timestream://Region=us-east-1;```
```

Author: Done

### DbVisualizer 
[Link to product webpage](https://www.dbvis.com/download/).

#### Adding the Amazon Timestream JDBC Driver
1. [Download](https://github.com/awslabs/amazon-timestream-driver-jdbc/releases/latest) the Timestream JDBC driver shaded JAR file (e.g., `amazon-timestream-jdbc-2.0.0-shaded.jar`)

> 2. Start the DbVisualizer application and navigate to the menu path: **Tools > Driver Manager...**. In the popped up window, click the "Custom" link on the right side.

Reviewer 2: Shall the user click on the "+" icon to add the driver?

Author: No. The "+" is only to add a driver from a template driver. We do not have a template driver so user could only add through "Custom" link.

Reviewer 2: Alternative option:
<img width="282" alt="image" src="https://user-images.githubusercontent.com/7035336/206814445-e796ea43-bb2f-449b-8f32-ca4f2e67c767.png">

Author: Ok, let me add it.

>     ![Example](../images/dbvisualizer/DbVisualizer1.png)

Reviewer 2: It would be good to highlight on the screenshots what the user should click on, so instructions are easy to use.

Author: Done

3. In the new popped up window, find **Driver Settings** tab and fill the following info.

    a. For the **Name:** field, enter **Timestream**

    b. For the **URL Format:** field, enter ```jdbc:timestream://PropertyName1=value1;PropertyName2=value2...```. For a list of connection properties, see [README](../../README.md#optional-connection-properties).

    c. Click the **+** on the right side to select the Timestream JDBC JAR file.

    d. Ensure the ```software.amazon.timestream.jdbc.TimestreamDriver``` is selected in the **Driver Class:** field. Your Driver Manager settings for **Timestream** should look like the following image.

    ![Example](../images/dbvisualizer/DbVisualizer2.png)

    e. Close the dialog. Now the Timestream JDBC driver is added and ready to use.

4. Connecting to Amazon Timestream Using DbVisualizer

    a. Navigate the menu path **Database > Create Database Connection**

    b. For the **Name** field, enter a descriptive name for the connection, for example ```TimestreamConnection```

    c. For the **Driver Type** field, ensure the **Timesteam** is selected.

>     d. For the **Database URL** field, enter your [JDBC connection string](../../README.md#optional-connection-properties). For example, ```jdbc:timestream://Region=us-east-1;```

Reviewer 2: Since it is easy to make an error, I would suggest adding a note that parameter names are case-sensitive.

Author: I think the README should be updated so this does not need to be mentioned everywhere.

Reviewer 2: Sounds good. Let's ensure it is updated.

Author: Done. In fact it is also mentioned in README.md. The better solution is to make it case insensitive which will be convenient for both users and developers.

    e. For the **Database Userid** field, enter your Amazon Timestream user ID.

    f. For the **Database Password** field, enter the corresponding password for the user ID.

    g. Your **Database Connection** dialog should look like the following.

    ![Example](../images/dbvisualizer/DbVisualizer3.png)

>     h. Then click **Connect** button to connect to Timestream.

Reviewer 2: Let's finish the instructions with the outcome/result. For example, mentioning that now you should be able to connect to the Timestream database, you have a success message, and you should see databases and tables on the left side as shown in the screenshot below.

Author: Done

    ![Example](../images/dbvisualizer/DbVisualizer2.png)

    e. Close the dialog. Now the Timestream JDBC driver is added and ready to use.

4. Connecting to Amazon Timestream Using DbVisualizer

    a. Navigate the menu path **Database > Create Database Connection**

    b. For the **Name** field, enter a descriptive name for the connection, for example ```TimestreamConnection```

    c. For the **Driver Type** field, ensure the **Timesteam** is selected.

    d. For the **Database URL** field, enter your [JDBC connection string](../../README.md#optional-connection-properties). For example, ```jdbc:timestream://Region=us-east-1;```

    e. For the **Database Userid** field, enter your Amazon Timestream user ID.

    f. For the **Database Password** field, enter the corresponding password for the user ID.

    g. Your **Database Connection** dialog should look like the following.

    ![Example](../images/dbvisualizer/DbVisualizer3.png)

    h. Click **Connect** button to connect to Timestream. Then you could connect to the Timestream database. You should be able to see all databases on the left side under the connection name as the example below.

>      ![Example](../images/dbvisualizer/DbVisualizer5.png)

Reviewer 2: Let's add a section with known limitations (issues we found during testing).

Author: Done
